app/flatpak-builtins-create-usb.c

```diff
@@ -71,6 +71,24 @@ commit_and_subpaths_new (const char *commit, const char * const *subpaths)
   return c_s;
 }
 
+static char **
```

Comment:
NItpick: I think it would be slightly better to return a `GPtrArray *` from this function, since then `commit_and_subpaths_new()` could use that and increment its ref count, rather than calling `g_strdupv()` on an array which has only just been allocated.

It’s a tiny optimisation (on arrays which are typically going to be empty or have only a few elements in; so a pointless tiny optimisation), and something not worth respinning this branch for — but maybe something to bear in mind for future branches?

Reply:
Yeah I can see how that would be better, but it would also require me to retest this, so it's not my priority right now